qemu/tests/cfg/timerdevice.cfg

```diff
@@ -9,15 +10,13 @@
             msr_tools_cmd = "a=$(rdmsr -d 0x00000010); echo -e ibase=16\\n $a  | tail -n 1 | while read n; do wrmsr 0x00000010 $(($n+100000000000)); echo $n; done"
         - clock_drift_with_sleep:
             only RHEL
-            no up
```

Comment:
In this case, vcpu needs to be pinned to specific host cpus, thus we need a `smp` host, I guess `no up` makes sense.